tonic/functional/drop_pixel.py

```diff
@@ -17,12 +17,15 @@ def identify_hot_pixel(events: np.ndarray, hot_pixel_frequency: float):
 
     assert "x" and "y" and "t" in events.dtype.names
 
+    if len(events)==0:
+        return []
+
```

Comment:
If an emty list is passed to the tonic transform, the functional call shouldn't be called in the first place. The check should probably happen somewhere here 
https://github.com/neuromorphs/tonic/blob/7455d0084710c95a4a6d56eed7f0c07cfaccd238/tonic/transforms.py#L230

Reply:
Solved